### DISCUSSION TITLE
How to prevent image unit conversion?

### DISCUSSION THREAD

**Discussion by ramiromagno** [quarto-dev/quarto-cli#6235] [Q&A] [answered]
### Description

Is there a simple way of preventing the `'9pc'` in:

https://github.com/ramiromagno/wiley-njd/blob/885a1827617952d3d01156ab68e6af8c8e118337/template.qmd#L131

getting converted to inches in latex:

```tex
\begin{figure}[t]
{\centering \includegraphics[width=\textwidth,height=1.5in]{empty.pdf}}
\caption{\label{fig-fig1}This is the sample figure caption.}
\end{figure}
```

The motivation is getting a latex template that is identical to the journal's provided template. These small changes, although not semantically different, do make an automatic file comparison between templates a lot more difficult.

**Comment by cderv** [comment]
This is Pandoc doing the conversion 
````
> quarto pandoc -t latex
![This is the sample figure caption.](empty){height='9pc'}
^Z
\begin{figure}
\centering
\includegraphics[width=\textwidth,height=1.5in]{empty}
\caption{This is the sample figure caption.}
\end{figure}
````

and I believe because `pc` is not a valid LaTeX unit. https://www.overleaf.com/learn/latex/Inserting_Images#Reference_guide

**Reply by ramiromagno** [reply]
It is less common, but valid: https://en.wikibooks.org/wiki/LaTeX/Lengths.

**Reply by cderv** [reply, marked as answer]
Learnt something thanks ! 

Then you should open an issue in Pandoc about this, so that this units is kept around for LaTeX outputs. https://github.com/jgm/pandoc/issues

**Comment by ramiromagno** [comment]
Done: https://github.com/jgm/pandoc/issues/8957.

**Comment by cderv** [comment]
It seems next Pandoc will support Point and Pica as units: https://github.com/jgm/pandoc/commit/ce6ef5cad0e9d05986aff4fe6ee00fa99b69ea9d
